File: C++/1046.cpp
#include <iostream>
#include <iomanip>
#include <cmath>
using namespace std;
 
int main() {
 
   int a , b, f;
   cin >> a >> b;
   f= a-b;
  if(a>=b){
      cout << "O JOGO DUROU " << (24 - a) + b << " HORA(S)" << endl;
  }
  else{
      cout << "O JOGO DUROU " <<   -a+b *pow(1,-1) << " HORA(S)" << endl;
  }
  

    return 0;

}

File: Python/1046.py
tempo = input().split()

x1 = int(tempo[0])
x2 = int(tempo[1])

if x1 == x2:
    duracao = 24
elif x1 < x2:
    duracao = x2 - x1
else:
    duracao = (24 - x1) + x2

print(f'O JOGO DUROU {duracao} HORA(S)')
